website/docs/configuration/reference.mdx

```diff
@@ -648,5 +648,105 @@ description: JSON schema reference documentation for GitProxy
 </blockquote>
 </details>
 
-----------------------------------------------------------------------------------------------------------------------------
+<details>
+<summary>
```

Comment:
@dcoric was this content written manually? Its supposed to be generated from the config schema, but I'm not seeing matching info added to the schema above. Hence, it will all be lost the next time that the ref doc is re-generated (`npm run gen-schema-doc` - requires you to have a python env setup for it to run in). I'm doing that on another PR that adds to the config schema so that will happen soon... A new PR is probably needed to correct this @JamieSlome @sam-holmes2